examples/cc3220/README.md

```diff
@@ -0,0 +1,134 @@
+Installing Dependencies & Related Documentation
+===============================================
```

Comment:
Readme.io is great because it's easier to collaborate on not only internally between you, Arpi and me, but also with our technical writer Jadwiga.  I think this document should actually be hosted there. Let me know if you don't have a readme.io account!